scripts/build/ios-appstore.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Builds the Firezone iOS client for submitting to the App Store
+
+set -euo pipefail
+
+source "./scripts/build/lib.sh"
+
+# Define needed variables
+app_profile_id=8da59aa3-e8da-4a8c-9902-2d540324d92c
+ne_profile_id=0fccb78a-97c0-41b9-8c54-9c995280ea8e
```

Comment:
These are vitally important to get right. Each time anything related to certs, capabilities, profiles, in the developer portal changes, these will change.